media-api/app/lib/ImageResponse.scala

```diff
@@ -225,8 +228,8 @@ class ImageResponse(config: MediaApiConfig, s3Client: S3Client, usageQuota: Usag
   def addValidity(valid: Boolean): Reads[JsObject] =
     __.json.update(__.read[JsObject]).map(_ ++ Json.obj("valid" -> valid))
 
-  def addUserCanEdit(userCanEdit: Boolean): Reads[JsObject] =
-    __.json.update(__.read[JsObject]).map(_ ++ Json.obj("userCanEdit" -> userCanEdit))
+  def addIsDownloadable(isDownloadable: Boolean): Reads[JsObject] =
+    __.json.update(__.read[JsObject]).map(_ ++ Json.obj("isDownloadable" -> isDownloadable))
```

Comment:
I'm not sure this is in line with the [argo-rest spec](https://github.com/argo-rest/spec) that media-api follows. I think the omission of a `download` key within [`links`](https://github.com/argo-rest/spec#links) should define if downloading is possible.

Reply:
Thanks @akash1810 - v useful